Mathlib/LinearAlgebra/AffineSpace/AffineMap.lean

```diff
@@ -49,8 +49,15 @@ induces a corresponding linear map from `V1` to `V2`. -/
 structure AffineMap (k : Type*) {V1 : Type*} (P1 : Type*) {V2 : Type*} (P2 : Type*) [Ring k]
   [AddCommGroup V1] [Module k V1] [AffineSpace V1 P1] [AddCommGroup V2] [Module k V2]
   [AffineSpace V2 P2] where
+  /-- The underlying function between the affine spaces `P1` and `P2`. -/
   toFun : P1 → P2
+  /-- The linear map between the corresponding vector spaces `V1` and `V2`.
+  This represents how differences of points under the affine map act. -/
   linear : V1 →ₗ[k] V2
+  /-- The key property that makes this an affine map: translating a point `p` by a vector `v`
+  and then applying the map is the same as applying the map to `p` and translating by
+  the image of `v` under the linear map. In other words, this shows that the map preserves
+  the affine space structure. -/
   map_vadd' : ∀ (p : P1) (v : V1), toFun (v +ᵥ p) = linear v +ᵥ toFun p
```

Comment:
I don't think docstrings on a proof field like this are very useful.

Reply:
Ok I've removed it.